tests/integrated/test-squash/runtest

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import netCDF4 as nc
+import itertools as it
+import os
+
+
+def run(cmd):
```

Comment:
`boututils` includes `launch_safe`, which this looks to be a reimplementation of. Is it possible to just use `launch_safe`?

Reply:
Yes, but is only 4 lines of code - thus basically as long as the other version. And for launch_safe I always need to check where to import it from, which is far more effort, without any gain, as far as I can see. Also, it make it slower. It also makes it harder to add timing instructions to the calls.
Again, if you feel strongly about this, I can change it, but I feel to see the advantage.

Reply:
It's four lines of code that need to be read and checked to see what they do by everyone reading this, versus something that's well integrated into our python library that already has a bunch of checks, documentation, handling for various systems, etc.

It took me 10 seconds to find what module `launch_safe` is in, I'm sure it would've taken you less time -- please reuse stuff rather than reinventing the wheel.

Reply:
launch_safe is also not what I want :-1: 
But as code is read more often then written, I will change ...